src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -32,21 +36,33 @@ export const MultiHopTrade = (props: CardProps) => {
   const methods = useForm({ mode: 'onChange' })
   const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`, { ssr: false })
   const { assetSearch } = useModal()
-  const sortedAssets = useAppSelector(selectAssetsSortedByMarketCapFiatBalanceAndName)
+  const buyAsset = useAppSelector(selectBuyAsset)
+  const sellAsset = useAppSelector(selectSellAsset)
+  const dispatch = useAppDispatch()
+  const setBuyAsset = useCallback(
+    (asset: Asset) => dispatch(swappers.actions.setBuyAsset(asset)),
+    [dispatch],
+  )
+  const setSellAsset = useCallback(
+    (asset: Asset) => dispatch(swappers.actions.setBuyAsset(asset)),
+    [dispatch],
+  )
+
+  const { supportedSellAssets, supportedBuyAssets } = useSupportedAssets()
+  const quotes = useGetTradeQuotes()
 
-  // TEMP: needs redux
-  const [sellAsset, setSellAsset] = useState(ETH)
-  const [buyAsset, setBuyAsset] = useState(FOX_MAINNET)
-  const supportedSellAssetsByMarketCap = sortedAssets
-  const supportedBuyAssetsByMarketCap = sortedAssets
-  const isSwapperInitialized = true
-  const swapperSupportsCrossAccountTrade = true
+  const lifiResult = quotes[SwapperName.LIFI]?.data
+
+  /* eslint-disable-next-line @shapeshiftoss/logger/no-native-console */
```

Comment:
Was this intended to be left in?

Reply:
yeah so much of the implementation is going to be console based until one of us gets around to implementing a UI

Reply:
Ah got it, in the context of this feature flag that makes sense.